auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -332,43 +330,32 @@ export default class AccountInfo extends Mixins(
   AccountChangeMixin,
   AccountMixin
 ) {
-  @CodesModule.State('suspensionReasonCodes')
-  private suspensionReasonCodes!: Code[]
-  @OrgModule.State('currentMembership') public currentMembership!: Organization
-  @OrgModule.State('currentOrgAddress') public currentOrgAddress!: Address
-  @OrgModule.State('permissions') public permissions!: string[]
-  @OrgModule.State('currentOrgPaymentType') public currentOrgPaymentType!: string
-  @userModule.State('currentUser') public currentUser!: KCUserProfile
-
-  @OrgModule.Getter('isBusinessAccount') public isBusinessAccount!: boolean
-  @OrgModule.Mutation('setCurrentOrganizationAddress')
-  public setCurrentOrganizationAddress!: (address: Address) => void
-
-  @OrgModule.Action('updateOrg') public updateOrg!: (
-    requestBody: CreateRequestBody
-  ) => Promise<Organization>
-
-  @OrgModule.Action('syncAddress') syncAddress!: () => Address
-  @OrgModule.Action('getOrgPayments') getOrgPayments!: () => any
-  @OrgModule.Action('updateOrganizationAccessType') updateOrganizationAccessType!: (
-    accessType: string
-  ) => Promise<Organization>
-
-  @OrgModule.Action('syncOrganization') syncOrganization!: (
-    currentAccount: number
-  ) => Promise<Organization>
-  @OrgModule.Action('suspendOrganization') suspendOrganization!: (
-    selectedSuspensionReasonCode: string
-  ) => Promise<Organization>
+  @State(useCodesStore) private suspensionReasonCodes!: Code[]
+  @State(useOrgStore) public currentMembership!: Organization
+  @State(useOrgStore) public currentOrgAddress!: Address
+  @State(useOrgStore) public permissions!: string[]
+  @State(useOrgStore) public currentOrgPaymentType!: string
+  @State(useUserStore) public currentUser!: KCUserProfile
```

Comment:
It's interesting that in our UIs, we never use the State like here. Aren't getters' sole purpose to get the state so we don't use it like this? I'm just asking Travis out of curiosity (no need to change anything).

Reply:
I don't see anything wrong with just using the state. I think Getters are created to read combinations of states

Reply:
Ahhh, that makes sense. Alright, sounds good!

Reply:
Also we aren't using the options store,  we're using the setup store so we don't use Getters.. one sec.. let me get the documentation

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/f97b34f5-4b31-4756-8f61-ea474b60e328)

Reply:
as you can see it's structured a bit different. Options Store - you don't need to build a reset function as well.

Reply:
Also the `pinia-class` will probably get axed as we move to `composition-api`

Reply:
Ohhh, OK. It's little bit different than with Options like you mentioned. I see. With Options Stores, we usually have a getter for each state property. Also, the state would have an interface to it.

I took a closer look at the stores here. In your case, there are no `state: `, `getters: `, and `actions: `. The format is much much different. Thanks for clearing that up Travis